<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务端</string>
  </property>
  <widget class="QLineEdit" name="portle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8888</string>
   </property>
   <property name="placeholderText">
    <string>端口号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>接收</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="rMessageText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="logList">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>161</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="sMessageText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boardBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>广播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始监听</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
